site_scons/buildutils.py

```diff
@@ -34,6 +36,373 @@
     TPathLike = TypeVar("TPathLike", Path, str)
 
 
+class Option:
+    """Object corresponding to SCons configuration option"""
+
+    def __init__(self,
+            name: str,
+            description: str,
+            default: Union[bool, str],
+            choices: list=None):
+        self.name = name
+        self.description = Option._deblank(description)
+        self.default = default
+        self.choices = choices
+
+        self._wrapper = textwrap.TextWrapper(width=80)
+        self.set_wrapper_indent(4)
+
+    @property
+    def parameters(self) -> tuple:
```

Comment:
Ideally, this could indicate the types in the tuple. Something like `"Union[Tuple[str, str, str, str], Tuple[str, str, str]]"` (I'm not sure about all the types).